### ISSUE TITLE
Trouble installing webdriver-manager

### ISSUE THREAD

**Issue by huguangju** [jakemmarsh/angularjs-gulp-browserify-boilerplate#178]
```js
> ./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update
sh: ./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager: No such file or directory
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "webdriver-update"
npm ERR! node v5.3.0
npm ERR! npm  v3.3.12
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! angularjs-gulp-browserify-boilerplate@1.5.6 webdriver-update: `./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the angularjs-gulp-browserify-boilerplate@1.5.6 webdriver-update script './node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angularjs-gulp-browserify-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update
npm ERR! You can get their info via:
npm ERR!     npm owner ls angularjs-gulp-browserify-boilerplate
npm ERR! There is likely additional logging output above.
```

> mac, node version: v5.3.0

**Comment by mnemanja** [comment]
Might be related to this? https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/issues/176

**Comment by karneaud** [comment]
Had to do my install manually. You should to

**Comment by specialorange** [comment]
@karneaud   I tried manually with `./node_modules/protractor/bin/webdriver-manager update` and several others.  I also tried `./node_modules/protractor/bin/webdriver-manager start` but got JAVA errors.   Can you expand on how to install manually? I am not familiar with selenium, and not exactly sure how this is relevant/required for this repo

**Comment by karneaud** [comment]
quite frankly what you explained is exactly what I did without the **JAVA errors**. Could it be your JAVA is out of date or inaccessible due to permissions?

**Comment by specialorange** [comment]
When running the `start` I get the following:
```
seleniumProcess.pid: 80567
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/openqa/grid/selenium/GridLauncher : Unsupported major.minor version 51.0
```
But I can't seem to find out how to run the selenium server, or how to point to a specific java version....

Running `java -version` I get
```
java version "1.6.0_65"
Java(TM) SE Runtime Environment (build 1.6.0_65-b14-468-11M4833)
Java HotSpot(TM) 64-Bit Server VM (build 20.65-b04-468, mixed mode
```
Which is the up to date (from what I have checked)

**Comment by mnemanja** [comment]
@specialorange I've managed to install the selenium manually by running `node ./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager update`

So, first `npm install` and then the manual webdriver-manager, and then it worked.

**Comment by specialorange** [comment]
I don't know what I am missing...
Yes I ran npm install, which I thought should install as many dependencies
as possible in the project's root `node_modules`'s folder.   But my
gulp-protractor node module doesn't have a `bin` folder, so I cant access
the updater.  Nor is it in the global NPM, or anywhere else.  I ran npm
update to get the latest info for node modules, but when I run `npm
install`, I still get the original error that this issue was opened up for `
npm ERR! Failed at the angularjs-gulp-browserify-boilerplate@1.5.6
webdriver-update script
'./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager
update'.
npm ERR! Make sure you have the latest version of node.js and npm
installed.` . Node -v is v5.8.0 and npm -v is 3.7.3​



Chris Frisina
724 I COLOR Orange
(724) 426-5676
